marconi-chain-index/src/Marconi/ChainIndex/Indexers/MintTokenEvent.hs

```diff
@@ -726,15 +745,94 @@ instance
 
 instance
   ( MonadIO m
+  , MonadError (Core.QueryError (QueryByAssetId MintTokenBlockEvents)) m
+  )
+  => Core.IsSync m event MintTokenEventIndexerCombine
+  where
+  lastSyncPoint idx = do
```

Comment:
Should this take the `lastSyncPoint`s from both indexers and take the minimum instead?